src/main/scala/com/research/nomad/markii/dataflow/AFTDomain.scala

```diff
@@ -423,6 +423,36 @@ case class AFTDomain(private val localNodeMap: Map[Local, AccessPath[AbsValSet[A
       d
     }
   }
+  def setDialogTitle(ctxMethod: SootMethod, stmt: Stmt, viewLocal: Local, param: Value): AFTDomain = {
+    var d = copy()
+    d = d.updateLocalViewNodes(ctxMethod, stmt, viewLocal,{
+      case viewNode: ViewNode =>
+        val newAttr = Set((AndroidView.ViewAttr.dialogTitle, param)).collect {
+          case (attr, stringConstant: StringConstant) =>
```

Comment:
nit: the logic here is a bit weird -- the `AndroidView.ViewAttr.dialogTitle` don't hvbe to be put in and then fetched out. I will probably merged it as it is anyway